evm_loader/cli/src/main.rs

```diff
@@ -1193,6 +1196,77 @@ fn command_neon_elf(
     });
 }
 
+fn command_neon_elf(
+    config: &Config,
+    program_location: Option<&str>,
+) -> CommandResult {
+    if let Some(program_location) = program_location {
+        // Reading program data from file
+        let program_data = read_program_data(program_location).unwrap();
+        let program_data = &program_data[..];
+        read_elf_parameters(config, program_data);
+        Ok(())
+    } else {
+        // Reading program data from account directly
+        if let Some(account) = config.rpc_client
+            .get_account_with_commitment(&config.evm_loader, config.commitment)?
+            .value
+        {
+            if account.owner == bpf_loader::id() || account.owner == bpf_loader_deprecated::id() {
+                read_elf_parameters(config, &account.data);
+                Ok(())
```

Comment:
You can also get rid of the following indentation just by returning Ok from here

Reply:
Fixed